### PR TITLE
Clean reusable blocks (Close #35)

### LINKED ISSUE #35: Clean up reusable blocks
The reusable blocks were imported from the previous platform and need some clean up:
* Inline the ones that are only used once
* Replace the tracker maintenance blocks with the `<Badges>` component
* See if there is anything else that does not make sense

### PR DESCRIPTION
**Pull request by rlh1994**
Cleaned up the reusable blocks:
- renamed any untitled ones that were required
- replaced old badged with new ones
- removed those only used in a single place
- Removed some badges entirely from older versions of trackers

I have left most of the JS ones as @mscwilson is looking to do unify those docs later anyway

### CODE REVIEW

**Review by matus-tomlein — APPROVED** [review]
That's a really nice clean-up! 💯

File: docs/collecting-data/collecting-from-own-applications/mobile-trackers/previous-versions/android-tracker/index.md [hunk 1]
```diff
@@ -1,11 +1,4 @@
...
 title: "Android Tracker v1.x"
-date: "2020-02-21"
 sidebar_position: 100
 ---
-
-```mdx-code-block
-import Block2895 from "@site/docs/reusable/untitled-reusable-block-35/_index.md"
-
-<Block2895/>
-```
```
Comment (stanch) [inline docs/collecting-data/collecting-from-own-applications/mobile-trackers/previous-versions/android-tracker/index.md:11]: Do these files become empty now?

File: docs/enriching-your-data/available-enrichments/index.md [hunk 2]
```diff
@@ -1,14 +1,41 @@
 ---
 title: "Available enrichments"
-date: "2020-02-15"
 sidebar_position: 20
 ---
 
-```mdx-code-block
-import EnrichmentList from "@site/docs/reusable/enrichment-list/_index.md"
+Snowplow offers a large number of enrichments that can be used to enhance your event data. An enrichment either updates or populates fields of the [atomic](https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow/atomic/jsonschema/1-0-0) event or adds a [self-describing](https://snowplowanalytics.com/blog/2014/05/15/introducing-self-describing-jsons/) context to [derived_contexts](https://github.com/snowplow/snowplow-scala-analytics-sdk/blob/2.1.0/src/main/scala/com.snowplowanalytics.snowplow.analytics.scalasdk/Event.scala#L165).
```
Comment (stanch) [inline docs/enriching-your-data/available-enrichments/index.md:6]:
```suggestion
Snowplow offers a large number of enrichments that can be used to enhance your event data. Most enrichments attach data to the event as [entities](...), although some enrichments fill or modify event fields directly.
```
Or something like that

File: docs/collecting-data/collecting-from-own-applications/mobile-trackers/previous-versions/android-tracker/index.md (hunk 1 above)
Comment (rlh1994) [reply docs/collecting-data/collecting-from-own-applications/mobile-trackers/previous-versions/android-tracker/index.md:11]: Good point, added docscards

**Pull request merged**